src/main/java/org/sonarlint/intellij/analysis/SonarLintAnalysisConfigurator.java

```diff
@@ -104,15 +102,15 @@ private static Charset getEncoding(Project p, @Nullable VirtualFile f) {
     return Charset.defaultCharset();
   }
 
-  private static List<InputFile> getInputFiles(Project p, ModuleRootManager moduleRootManager, Collection<VirtualFile> filesToAnalyze) {
+  private static List<ClientInputFile> getInputFiles(Project p, ModuleRootManager moduleRootManager, Collection<VirtualFile> filesToAnalyze) {
     Collection<String> testFolderPrefix = findTestFolderPrefixes(moduleRootManager);
-    List<InputFile> inputFiles = new LinkedList<>();
+    List<ClientInputFile> inputFiles = new LinkedList<>();
 
     for (VirtualFile f : filesToAnalyze) {
       boolean test = isTestFile(testFolderPrefix, f);
       Path path = Paths.get(f.getPath());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "path". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1854)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "path" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1481)